tests/pytorch/nightly/common.libsonnet

```diff
@@ -20,27 +20,16 @@ local volumes = import 'templates/volumes.libsonnet';
 
 local version = 'nightly';
 {
-  PyTorchTest:: common.PyTorchTest {
-    frameworkPrefix: 'pt-%s' % version,
+  local Nightly = {
+    frameworkPrefix: 'pt-nightly',
```

Comment:
`version` was set to `nightly` above. What's difference between that and harding it to 'pytorch-nightly'>

Reply:
I think the `version` var was added to make it easier to substitute in a release version for `frameworkPrefix`, `softwareVersion`, and `imageTag`. That didn't really work out since these tags all have subtly different formats; see `common.libsonnet` for r1.10:

https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/28ee676a0b56c1adf1fc1505b0c1ef214301b862/tests/pytorch/r1.10/common.libsonnet#L20-L41

To clarify, the net effect of my changes in this file is that the `imageTag` default changes from `nightly` to `nightly_3.7`. I also saw the opportunity to refactor and reduce repetition in this file. I double-checked the generated diffs to make sure the changes to `frameworkVersion` and `softwareVersion` has no effect.